java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -258,8 +258,8 @@ public Result getDriverPath(Capabilities options, boolean offline) {
     List<String> arguments = new ArrayList<>();
     arguments.add("--browser");
     arguments.add(options.getBrowserName());
-    arguments.add("--language-binding");
-    arguments.add("java");
+    //    arguments.add("--language-binding");
```

Comment:
Is this intentionally commented out?

Reply:
Yes, Selenium Manager work was in progress and locally I commented it out to ensure I run tests. I think it is fixed now.